hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/props/MirrorFreezeStatus.java

```diff
@@ -0,0 +1,17 @@
+package com.hedera.mirror.test.e2e.acceptance.props;
+
+public enum MirrorFreezeStatus {
+
+    NOT_APPLICABLE("NOT_APPLICABLE"),
+
+    FROZEN("FROZEN"),
+
+    UNFROZEN("UNFROZEN");
+
+
+    private String value;
+
+    MirrorFreezeStatus(String value) {
+        this.value = value;
+    }
+}
```

Comment:
```suggestion
public enum MirrorFreezeStatus {
    NOT_APPLICABLE,
    FROZEN,
    UNFROZEN
}
```

same for the other enum